loadtesting/k6/tests/utils/ethClient.js

```diff
@@ -38,13 +35,19 @@ export function sendTokens(client, from, to, value, gas, gasPrice, nonce) {
         "chain_id": client.chainID,
     };
 
-    if (transaction["gas"] == 0) {
-        transaction["gas"] = client.estimateGas(transaction);
+    if (gas == null) {
+        transaction["gas"] = client.estimateGas({
```

Comment:
why don't you use transaction object here as it was?

Reply:
you're right, it was just old debug code
fixed